### PR TITLE
Add a graphql endpoint to construct the clientmount command.

### PR DESCRIPTION
**Pull request by johnsonw**
Signed-off-by: johnsonw <wjohnson@whamcloud.com>

---
This change is [Reviewable](https://reviewable.io/reviews/whamcloud/integrated-manager-for-lustre/2392)

### CODE REVIEW

**Comment by johnsonw** [comment]
This shows that the gui is now making an asynchronous call to graphql to retrieve the client mount command. The result is being displayed as follows:

image

**Comment by johnsonw** [comment]
image

**Comment by johnsonw** [comment]
I'm now in the process of adding this call to the manager cli.

**Review by jgrund — CHANGES_REQUESTED** [review]
The client mount in the GUI should refresh itself periodically, maybe once a minute.

**Comment by johnsonw** [comment]
Fetching every 60 seconds now:
image

File: iml-gui/crate/src/page/filesystem.rs [hunk 1]
```diff
@@ -79,10 +85,45 @@ pub fn init(cache: &ArcCache, model: &mut Model, orders: &mut impl Orders<Msg, G
     stratagem::init(cache, &model.stratagem, &mut orders.proxy(Msg::Stratagem));
 
     orders.send_msg(Msg::FetchStats);
+
+    orders.send_msg(Msg::FetchMountCommand);
+}
+
+async fn delay_and_fetch_mount_command(delay: u32) -> Result<Msg, Msg> {
```
Comment (jgrund) [inline iml-gui/crate/src/page/filesystem.rs:92]: use `sleep_with_handle` instead.

Comment (johnsonw) [reply iml-gui/crate/src/page/filesystem.rs:92]: Sure. When we upgrade to 0.7 (or even 0.8) we can use streams on the orders object :)

Comment (jgrund) [reply iml-gui/crate/src/page/filesystem.rs:92]: yup

**Review by jgrund — APPROVED** [review]

**Pull request merged**